### PR TITLE
Add reposync configuration settings to tshoot-sync

### PR DESCRIPTION
**Pull request by agraul**
# Description

I introduce two configuration values to control reposync timeout and minimal transfer rate in https://github.com/uyuni-project/uyuni/pull/2735/ and want them to be documented.

# Target branches

Which documentation version does this PR apply to?

TODO: check if backports are needed

- [x] Master (Default)
- [x] Manager-4.1
- [ ] Manager-4.0

### CODE REVIEW

File: modules/administration/pages/tshoot-sync.adoc
Comment (Loquacity) [inline modules/administration/pages/tshoot-sync.adoc:78]:
```suggestion
You can resolve this error by specifying [literal]``reposync_timeout`` and [literal]``reposync_minrate`` configuration values in [path]``/etc/rhn/rhn.conf``. 
By default, when less than 1000 bytes per second are transferred in 300 secs, the download is aborted.
You can adjust the number of bytes per second with [literal]``reposync_minrate``, and the number of seconds to wait with [literal]``reposync_timeout``.
```

Just trying to make that second sentence a bit clearer, and also adjust the newline syntax to match the rest of the doc.

**Review by keichwa — APPROVED** [review]

**Comment by Loquacity** [comment]
I'm happy with this, will approve when you mark it ready for review.

**Comment by Loquacity** [comment]
@agraul Can you please mark this as ready for review? That way we can merge.

**Comment by agraul** [comment]
@Loquacity will do as soon as the code change is approved & merged

**Comment by Loquacity** [comment]
ah ha, I didn't notice that. Thanks!

**Comment by agraul** [comment]
@Loquacity We've decided to backport the change to 4.1(merged already), what's the process for backporting this documentation PR?

**Comment by Loquacity** [comment]
> @Loquacity We've decided to backport the change to 4.1(merged already), what's the process for backporting this documentation PR?

:heavy_check_mark: 4.1 https://github.com/uyuni-project/uyuni-docs/pull/889

**Pull request merged**
